Procfile

```diff
@@ -0,0 +1,3 @@
+server1: bin/rails s --port=3001 --pid=tmp/pids/server1.pid
```

Comment:
Should I use `falcon serve` instead of `bin/rails s`?

Reply:
Yes, `bin/rails s` uses Rackup which is very sub-optimal.